avahi-core/server.c

```diff
@@ -1295,7 +1295,11 @@ static void update_fqdn(AvahiServer *s) {
 }
 
 int avahi_server_set_host_name(AvahiServer *s, const char *host_name) {
-    char *hn = NULL;
+    char label_escaped[AVAHI_LABEL_MAX*4+1];
```

Comment:
I think it should be a separate constant and it should be `(AVAHI_LABEL_MAX-1)*4+1` but it can be changed later. It's getting messy though
```sh
avahi-common/alternative.c:    char label[AVAHI_LABEL_MAX], alternative[AVAHI_LABEL_MAX*4+1];
avahi-common/domain.c:    char escaped_name[AVAHI_LABEL_MAX*4];
avahi-core/server.c:    char label_escaped[AVAHI_LABEL_MAX*4+1];
```